modules/integration/src/test/scala/zio/elasticsearch/HttpExecutorSpec.scala

```diff
@@ -1407,11 +1407,29 @@ object HttpExecutorSpec extends IntegrationSpec {
             Executor.execute(ElasticRequest.createIndex(firstSearchIndex)),
             Executor.execute(ElasticRequest.deleteIndex(firstSearchIndex)).orDie
           ),
+          test("search for a document using a simple query string query") {
+            checkOnce(genDocumentId, genTestDocument, genMultiWordString(), genDocumentId, genTestDocument) {
+              (firstDocumentId, firstDocument, multiWordString, secondDocumentId, secondDocument) =>
+                val firstDoc = firstDocument.copy(stringField = multiWordString)
+
+                for {
+                  _ <- Executor.execute(ElasticRequest.upsert(firstSearchIndex, firstDocumentId, firstDoc))
+                  _ <- Executor.execute(
+                         ElasticRequest.upsert(firstSearchIndex, secondDocumentId, secondDocument).refreshTrue
+                       )
+                  searchTerm = multiWordString.split("\\s+").head
+                  query      = simpleQueryString(searchTerm).fields(Chunk(TestDocument.stringField))
```

Comment:
Can you provide a test with "empty" fields just to make sure that Elasticsearch is searching for all fields then instead?